tests/Unit/Core/Localization/CLDR/DataLayer/CurrencyCacheTest.php

```diff
@@ -58,32 +58,20 @@ protected function setUp(): void
     public function testReadWrite()
     {
         $data = new CldrCurrencyData();
-        /* @phpstan-ignore-next-line */
-        $data->foo = ['bar', 'baz'];
```

Comment:
I'm a bit worried about this one, seems like the test explicitly tests out dynamic properties I know they are not allowed anymore in PHP 8.2 but then are we sure our implementation doesn't need them? If the tests validate this behaviour it may be because it is needed, so regardless of the tests themselves maybe we need to make sure this doesn't have impact on the CLDR no?

Reply:
This is only to test getting data and fetching data return the same results (like a in-memory cache).